package.json

```diff
@@ -39,29 +41,29 @@
     "@types/jest": "^29.5.12",
     "@types/node": "20.12.12",
     "@types/ramda": "^0.28.25",
-    "@types/react": "^18.3.1",
+    "@types/react": "^18.3.3",
```

Comment:
There is some weird sub dep that breaks the react types, which makes docs fail to build. I need to investigate it and pin naughty package to specific version.